app/cdap/text/text-en.yaml

```diff
@@ -3432,8 +3432,22 @@ features:
       success: You have successfully created the pipeline "{pipelineName}".
     Skip: Skipped
   WranglerNewUI:
+    BreadCrumb:
+      labels:
+        dataSources: Data Sources
+
     NoRecordScreen:
       gridTable:
         title: No rows left in this sample
         subtitle: Remove some recipe steps to view some data
+    Snackbar:
+      labels:
+        retrieveFailure: Failed to retrieve sample data for
+        workspaceFailure: Failed to create workspace for
+        datasetSuccess: Dataset loaded successfully 
+        sampleFailure: Failed to retrieve sample data
+        success: Success
+        failure: Failure
+    
```

Comment:
keep only 1 extra line at the bottom of the page

Reply:
Done